torch/_meta_registrations.py

```diff
@@ -2,6 +2,14 @@
 
 meta_lib = torch.library.Library("aten", "IMPL", "Meta")
 
+def toRealValueType(dtype):
```

Comment:
This is totally fine and there's no need to change it, but I wonder how many definitions of this function we have in Python:

https://github.com/pytorch/pytorch/blob/3e92cae0dd11f57eb5773e9a7ae87fa0698997a5/torch/_prims/utils.py#L421

Reply:
We need to put this on the torch.dtype object, IMO